cmd/sidecar/main.go

```diff
@@ -40,17 +46,33 @@ func main() {
 		os.Exit(1)
 	}
 
-	configSecrets, err := secretsConfig.NewFromEnv(tokenFilePath)
+	configStorage, err := storageConfig.NewFromEnv()
 	if err != nil {
 		errLogger.Printf(err.Error())
 		os.Exit(1)
 	}
+	// Create new Storage
+	stor, err := storage.NewStorageHandler(*configStorage)
+
+	//if conjurSecretsDest == 1  &&  authn.Config.ContainerMode != "init" {
+	//      // notify on invalid configuration and exit
+	//      errLogger.Printf(" appropriate message for not supporting sidecar ....")
+	//      os.Exit(1)
+	//}
+
+	if conjurSecretsDest == 1 {
+		configSecrets, err := secretsConfig.NewFromEnv(tokenFilePath)
+		if err != nil {
+			errLogger.Printf(err.Error())
```

Comment:
please some text here so we can pinpoint it easier in the future. for example:
`errLogger.Printf("failed to create secrets config: %s", err.Error())`

Reply:
added but I do think we should keep it as general as possible because this could change when we actually move these code blocks to their files